plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -569,6 +569,8 @@ protected function handle_row_actions( $item, $column_name, $primary ) : string
 			);
 		}
 
+		$actions = apply_filters( 'comment_row_actions', array_filter( $actions ), $item );
```

Comment:
@sackadelic Could you add a docblock comment on the line above to clarify the meaning of the filter? This is required to pass the CI check and would be helpful for other developers who use the code.